nixos/modules/services/networking/hostapd.nix

```diff
@@ -20,12 +20,13 @@ let
     ssid=${cfg.ssid}
     hw_mode=${cfg.hwMode}
     channel=${toString cfg.channel}
+    ${optionalString (cfg.countryCode != null) ''country_code=${cfg.countryCode}''}
```

Comment:
If the country code is set we could also set `ieee80211d=1` to announce the country IE in beacons, which let's clients reconfigure their radio accordingly.

```
# Enable IEEE 802.11d. This advertises the country_code and the set of allowed
# channels and transmit power levels based on the regulatory limits. The
# country_code setting must be configured with the correct country for
# IEEE 802.11d functions.
# (default: 0 = disabled)
#ieee80211d=1
```

Reply:
Added on extra commit